content/references/http-websocket-apis/public-api-methods/clio-methods/nft_info.md

```diff
@@ -0,0 +1,135 @@
+---
+html: nft_info.html
+parent: clio-methods.html
+blurb: Retrieve information about the specified NFT using Clio server's `nft_info` API.
+labels:
+  - Non-fungible Tokens, NFTs
+---
+# nft_info
+[[Source]](https://github.com/XRPLF/clio/blob/4a5cb962b6971872d150777881801ce27ae9ed1a/src/rpc/handlers/NFTInfo.cpp "Source")
+
+The `nft_info` command asks the Clio server for information about the [NFT](non-fungible-tokens.html) being queried. 
+
+## Request Format
+An example of the request format:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+  "id": 1,
+  "command": "nft_info",
+  "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C"
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+    "method": "nft_info",
+    "params": [
+      {
+          "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C"
+      }
+    ]
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+<!-- To DO: Add an example command to the assets/js/apitool-methods-ws.js file. The WebSocket Tool requires access to a publicly available Clio server.
+[Try it! >](websocket-api-tool.html#nft_info)-->
```

Comment:
For what it's worth, I put together a test commit ( https://github.com/mDuo13/xrpl-dev-portal/commit/735656cedb93405770bb4eb52694f8f18174cd9b ) that (a) adds the nft_info command to the WS tool methods, and and (b) adds a public Clio server on NFT-Devnet to the tool. This is **just an example** to show loosely what needs to be done on the tool side to make the "Try it!" work. I don't think we are sure that this particular Clio URL will be long-term supported, though, so **do not merge** my commit.